aries-core/src/js/components/core/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Box } from 'grommet';
+
+export const Sidebar = ({ children, header, ...rest }) => {
+  return (
+    <Box pad="small" gap="large" height={{ min: '100%' }} {...rest}>
+      {header}
+      <Box justify="between" align="center" fill="vertical">
```

Comment:
Is there a difference between using `fill="vertical"` and `flex="grow"` here? I am only asking because my thought would've been to use `flex="grow"`

Reply:
I think that `flex="grow"` means that the element can grow as long as it has available space, while we want the element to spread only vertically and take the full 100% height.